src/App.tsx

```diff
@@ -1,125 +1,149 @@
+import NotFound from "./Features/Error/NotFound";
+import Loading from "./Features/Loading";
+import styles from "./App.module.scss";
 
 const defaultTimeout = 10000;
 
 const WordSearch = Loadable({
-  loader: () => import('./Features/WordSearch'),
+  loader: () => import("./Features/WordSearch"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const AutoConvert = Loadable({
-  loader: () => import('./Features/Encoding/AutoConvert'),
+  loader: () => import("./Features/Encoding/AutoConvert"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Autokey = Loadable({
-  loader: () => import('./Features/Cipher/Autokey'),
+  loader: () => import("./Features/Cipher/Autokey"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Braille = Loadable({
-  loader: () => import('./Features/Encoding/Braille'),
+  loader: () => import("./Features/Encoding/Braille"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Caesar = Loadable({
-  loader: () => import('./Features/Cipher/Caesar'),
+  loader: () => import("./Features/Cipher/Caesar"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Character = Loadable({
-  loader: () => import('./Features/Encoding/Character'),
+  loader: () => import("./Features/Encoding/Character"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Home = Loadable({
-  loader: () => import('./Features/Home'),
+  loader: () => import("./Features/Home"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Morse = Loadable({
-  loader: () => import('./Features/Encoding/Morse'),
+  loader: () => import("./Features/Encoding/Morse"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Nato = Loadable({
-  loader: () => import('./Features/Encoding/Nato'),
+  loader: () => import("./Features/Encoding/Nato"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const NavalFlag = Loadable({
-  loader: () => import('./Features/Encoding/NavalFlag'),
+  loader: () => import("./Features/Encoding/NavalFlag"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Pigpen = Loadable({
-  loader: () => import('./Features/Pigpen'),
+  loader: () => import("./Features/Pigpen"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Resistor = Loadable({
-  loader: () => import('./Features/Resistor'),
+  loader: () => import("./Features/Resistor"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Semaphore = Loadable({
-  loader: () => import('./Features/Encoding/Semaphore'),
+  loader: () => import("./Features/Encoding/Semaphore"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Settings = Loadable({
-  loader: () => import('./Features/Settings'),
+  loader: () => import("./Features/Settings"),
   loading: Loading,
   timeout: defaultTimeout,
 });
 
 const Vigenere = Loadable({
-  loader: () => import('./Features/Cipher/Vigenere'),
+  loader: () => import("./Features/Cipher/Vigenere"),
```

Comment:
Nit: In the future, it would be nice to have formatting fixes as a separate change from functional refactors like this to make it obvious what the actual delta is

Reply:
Yeah, sorry about that. This change started off rather small and spiraled out of control. :(